packages/ui/src/Tooltip/Tooltip.tsx

```diff
@@ -34,38 +26,43 @@ export interface TooltipProps {
     className?: string;
 }
 
-interface State {
-    tooltipIsOpen: boolean;
-}
-
 /**
- * Use Tooltip component to display a list of choices, once the handler is triggered.
+ * @deprecated This component is deprecated and will be removed in future releases.
+ * Please use the `<Tooltip />` component from the `@webiny/admin-ui` package instead.
  */
-class Tooltip extends React.Component<TooltipProps, State> {
-    public override state = {
-        tooltipIsOpen: false
-    };
+export const Tooltip = ({
+    children,
+    content,
+    placement,
+    className = "webiny-ui-tooltip"
+}: TooltipProps) => {
+    const mapPlacementToSideAndAlign = useCallback((placement?: string) => {
```

Comment:
This could've actually also been done with a `useMemo`, so the `side` and `align` are immediately returned. With this change:

1. there would be no need to do a function call below
2. would be also more performant

Reply:
Good point, thank you!